operators/src/processing/temporal_raster_aggregation/subquery.rs

```diff
@@ -44,16 +44,24 @@ where
     P: Pixel,
     F: TemporalRasterPixelAggregator<P> + 'static,
 {
-    pub fn add_tile(&mut self, in_tile: RasterTile2D<P>) -> Result<()> {
-        self.time = self.time.union(&in_tile.time)?;
+    pub fn add_tile(&mut self, in_tile: RasterTile2D<P>) {
+        // we do no time modification here since the result represents the aggreate over query time
```

Comment:
this comment doesn't really make sense without knowing that we erroneously did it before right? Maybe instead describe what the method does here? 

Perhaps something like this:

"Add a tile to the accumulator, which represents the aggregate over a time interval that contains the in_tile. TODO: for tiles which are only partially contained in the aggregate time interval, investigate whether the pixels have to be scaled (e.g. if the pixel is a count, assume uniform distribution and divide it by the fraction of time that is contained in the aggregate)".

Reply:
[eebc485](https://github.com/geo-engine/geoengine/pull/769/commits/eebc485ccf9b5342e0401e74ca701799763a3a39)